v2/api-validator/src/server/app.ts

```diff
@@ -27,10 +27,26 @@ export class WebApp {
       logger: log.pinoLogger,
       schemaErrorFormatter,
     });
+
+    // For POST routes that do not define request body, allow any content type
+    // and ignore the body
+    const routesWithoutBodyContentParser = (req, body, done) => {
+      if (!req.routeSchema.body) {
+        done(null, {});
+      } else {
+        done(new ContentTypeError(req.headers.contentType), undefined);
+      }
+    };
+    this.app.addContentTypeParser(
+      'application/x-www-form-urlencoded',
+      routesWithoutBodyContentParser
+    );
```

Comment:
Is it expected that post requests with no body will be sent with this content type?
If it is simulated this way, it should be expected that the real client will also work the same way.

Reply:
It just means that we do not require any specific content type in this case. Our client for some reason chooses to use this one 🤷‍♂️